libraries/conf_helpers.rb

```diff
@@ -46,7 +46,8 @@ def self.parse_config(conf_body)
         case ln
         when /^\s*\[([^#]+)\]\s*/
           current_section = Regexp.last_match[1]
-        when /^\s*([^#]+?)\s*=\s*([^#]+?)\s*(?:#.*)?$/
+          current_config[current_section] ||= {}
+        when /^\s*([^#]+?)\s*=\s*([^#\v]*?)\s*(?:#.*)?$/
```

Comment:
It was capturing the newline in the values, which lead to weird things like "someKey = \n". I added some unit tests for this case.